Updated Raw HID docs to clarify packet/report length (#11211)
qmk/qmk_firmware: docs/feature_rawhid.md

}
```

> These two functions can receive and send respectively packets from and to the host with length `RAW_EPSIZE` bytes (32 on LUFA/ChibiOS/V-USB, 64 on ATSAM).

Reviewer: ```suggestion
These two functions send and receive packets of length `RAW_EPSIZE` bytes to and from the host (32 on LUFA/ChibiOS/V-USB, 64 on ATSAM).
```
Sorry, I realised this sentence was a little awkward.

> `raw_hid_receive` can receive variable size packets from host with maximum length `RAW_EPSIZE` and minimum length of 32 bytes. `raw_hid_send` on the other hand can send packets to host of exactly `RAW_EPSIZE` length, therefore it should be used with data of length `RAW_EPSIZE`.

Reviewer: ```suggestion
These two functions can receive and send respectively packets from and to the host with length `RAW_EPSIZE` bytes (32 on LUFA/ChibiOS/V-USB, 64 on ATSAM).
```
This entire page kinda needs a bit of rewriting, tbh.
